README.md

```diff
@@ -87,6 +87,15 @@ To allow for extra properties, use `.toMatchTypeOf`. This checks that an object
 expectTypeOf({a: 1, b: 1}).toMatchTypeOf({a: 1})
 ```
 
+`.toEqualTypeOf` and `.toMatchTypeOf` both fail on missing properties:
+
+```typescript
+// @ts-expect-error
+expectTypeOf({a: 1}).toEqualTypeOf({a: 1, b: 1})
+// @ts-expect-error
+expectTypeOf({a: 1}).toMatchTypeOf({a: 1, b: 1})
+```
```

Comment:
To be clear, could this also be written as?

```typescript
expectTypeOf({a: 1}).not.toEqualTypeOf({a: 1, b: 1})
expectTypeOf({a: 1}).not.toMatchTypeOf({a: 1, b: 1})
```

Reply:
Yes - at this stage of the "readme", `.not` hasn't been introduced yet so I opted for `// @ts-expect-error` since it makes very clear what a failing assertion looks like.

Also, I added a brief explanation of the slightly unusual way that the readme is generated - these tests _are_ real tests, but they are also the documentation for the library: https://github.com/mmkal/expect-type#contributing